backend/git_real/permissions.py

```diff
@@ -22,6 +22,25 @@ def has_permission(self, request, view):
         correct = recieved_digest == "sha1=" + digest
         if not correct:
             return False
-            # breakpoint()
 
         return correct
+
+def user_can_see_repository(repo, user):
```

Comment:
This is only used in one class, right? 

Why not put it in that class? Or are you intending it to be used elsewhere as well?